From 2fd3185f5de5c59fc5123ac0482eb5bcb42eb197 Mon Sep 17 00:00:00 2001
From: Tanu Kaskinen <tanu.kaskinen@linux.intel.com>
Date: Sat, 3 May 2014 14:10:52 +0300
Subject: [PATCH 06/31] bluetooth: Remove redundant assignments

---
 src/modules/bluetooth/bluez5-util.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/modules/bluetooth/bluez5-util.c b/src/modules/bluetooth/bluez5-util.c
index 7eecc18..8498c42 100644
--- a/src/modules/bluetooth/bluez5-util.c
+++ b/src/modules/bluetooth/bluez5-util.c
@@ -419,8 +419,6 @@ static void device_free(pa_bluetooth_device *d) {
     if (d->uuids)
         pa_hashmap_free(d->uuids);
 
-    d->discovery = NULL;
-    d->adapter = NULL;
     pa_xfree(d->path);
     pa_xfree(d->alias);
     pa_xfree(d->address);
-- 
1.9.3

