From 3f56a1195c67734ed8ea0a0bdd862a22f6eb9d52 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Fri, 14 Nov 2014 13:07:48 +0100
Subject: [PATCH 31/31] backend-native: fix commands

---
 src/modules/bluetooth/backend-native.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/modules/bluetooth/backend-native.c b/src/modules/bluetooth/backend-native.c
index aa2b756..3def379 100644
--- a/src/modules/bluetooth/backend-native.c
+++ b/src/modules/bluetooth/backend-native.c
@@ -248,7 +248,7 @@ static void rfcomm_io_callback(pa_mainloop_api *io, pa_io_event *e, int fd, pa_i
 
         pa_log_debug("RFCOMM >> OK");
 
-        len = write(fd, "\r\nOK\r\n", 5);
+        len = write(fd, "\r\nOK\r\n", 6);
 
         /* we ignore any errors, it's not critical and real errors should
          * be caught with the HANGUP and ERROR events handled above */
@@ -285,7 +285,7 @@ static void set_speaker_gain(pa_bluetooth_transport *t, uint16_t gain) {
 
     t->speaker_gain = gain;
 
-    len = sprintf(buf, "+VGS=%d\r\n", gain);
+    len = sprintf(buf, "\r\n+VGS=%d\r\n", gain);
     pa_log_debug("RFCOMM >> +VGS=%d", gain);
 
     written = write(trfc->rfcomm_fd, buf, len);
@@ -304,7 +304,7 @@ static void set_microphone_gain(pa_bluetooth_transport *t, uint16_t gain) {
 
     t->microphone_gain = gain;
 
-    len = sprintf(buf, "+VGM=%d\r\n", gain);
+    len = sprintf(buf, "\r\n+VGM=%d\r\n", gain);
     pa_log_debug("RFCOMM >> +VGM=%d", gain);
 
     written = write (trfc->rfcomm_fd, buf, len);
-- 
1.9.3

